docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,507 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "\n",
+    "from qiskit_experiments.test.t2hahn_backend import T2HahnBackend\n",
+    "# FakeJob is a wrapper for the backend, to give it the form of a job\n",
+    "from qiskit_experiments.test.utils import FakeJob\n",
+    "\n",
+    "\n",
+    "estimated_t2hahn2 = 20\n",
+    "# The behavior of the backend is determined by the following parameters\n",
+    "backend2 = T2HahnBackend(\n",
+    "    t2hahn=[20],\n",
+    "    frequency=[100100],\n",
+    "    initialization_error=[0.0],\n",
+    "    readout0to1=[0.02],\n",
+    "    readout1to0=[0.02],\n",
+    "    conversion_factor=conversion_factor,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "dt_factor2 = apply_prefix(1, unit2)\n",
```

Comment:
You're not using `dt_factor2`, remove this line.

Reply:
removed `dt` factors.